docker-compose.yml

```diff
@@ -406,4 +399,3 @@ services:
 
 networks:
   flowbroker:
-    name: ${FLOWBROKER_NETWORK}
```

Comment:
Someone (@raulnegreiros) needs to check this line in particular - there was a reason behind this setting.

Reply:
yes, it is important... flowbroker uses this network to communicate with the remote nodes

Reply:
I'll check how to do that in the docker-compose version 3. Because it's a little different.